docs/Splashkit/Applications/Build a COOL game/DiceyCombat/diceycombat.cpp

```diff
@@ -0,0 +1,279 @@
+#include "splashkit.h"
+#include "all the headers" 
+
```

Comment:
If this is pseudocode I would make it a .txt or .md file so it is clear

At the moment it looks like broken code.